### ISSUE TITLE
Separate color for a path prefix

### ISSUE THREAD

**Issue by ktosiek** [joseph-jalbert/colortabs#4]
It would be useful to be able to restrict the color to a specific URL path, not just whole domain.

My use case is: I have some applications that I use every day, which have dedicated control panels on the same domain. I want a neutral color for most pages on the domain, but I'd like everything under /redaction/ on the production instance to be red.

**Comment by mikko001** [comment]
I agree. I have multiple instances of an application on one domain. Would be great to be able to define: 128.0.0.1/app1 -> green, 128.0.0.1/app2 -> red, etc...

**Comment by joseph-jalbert** [comment]
Good idea, @ktosiek & @mikko001, I am going to implement this in the next version. I will let you know when it is ready.

**Comment by mikko001** [comment]
Great! You probably noticed that I created a pull request for a simple solution.

**Comment by joseph-jalbert** [comment]
Thanks! I'm very busy right now but should have some time to review it over the weekend
edit: PR was reviewed and closed
